k8s/helm-operator/crd.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: valdreleases.vald.vdaas.org
```

Comment:
release.vald.vdaas.org ?

Reply:
This must be `spec.names.plural` + `spec.group`.